guides/doc-Provisioning_Guide/topics/Virt-RHV.adoc

```diff
@@ -223,9 +247,9 @@ To create a host with image-based provisioning, use the `hammer host create` com
 
 [options="nowrap" subs="+quotes"]
 ----
-# hammer host create --name "rhv-test2" --organization "_My_Organization_" \
+# hammer host create --name "ovirt-test2" --organization "_My_Organization_" \
```

Comment:
Why not use the variable?

Reply:
Hostnames are always lowercase, we do not enforce this tho. Probably not a problem, it should be converted to lowercase automatically.

Reply:
What I mean is that this is now going to display as ovirt for Satellite also, and I think it should be rhv-test